tests/integration/test_stampipes.py

```diff
@@ -3,8 +3,19 @@
 from pathlib import Path
 
 
-@pytest.mark.workflow(name='test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads')
-def test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads_bams_match(test_data_dir, workflow_dir, bams_match):
-    actual_bam_path = workflow_dir / Path('test-output/flagged.bam')
-    expected_bam_path = test_data_dir / Path('dnase/alignment/flagged.bam')
+@pytest.mark.workflow(name="test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads")
+def test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads_bams_match(
+    test_data_dir, workflow_dir, bams_match
+):
+    actual_bam_path = workflow_dir / Path("test-output/flagged.bam")
+    expected_bam_path = test_data_dir / Path("dnase/alignment/flagged.bam")
     assert bams_match(actual_bam_path.as_posix(), expected_bam_path.as_posix())
+
+
+@pytest.mark.workflow(name="test_get_chrombuckets")
+def test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads_bams_match(
```

Comment:
One line and need to rename this.

Reply:
good catch. Sure, I need to turn off my autoblack for this..

Reply:
kinda would be fan of autoformatting..